README.md

```diff
@@ -84,8 +84,8 @@ Example:
 
 ....
 
-const user = user.fakeOnly('name');
-const user = user.fakeOnly('name', 'age');
+const user = User.fakeOnly('name');
```

Comment:
@stevesweets This are instance level methods to let update `user` instance with changes.